dynatrace/build.gradle

```diff
@@ -44,6 +47,15 @@ dependencies {
 
     testImplementation("org.mockito:mockito-core:3.12.4")
     testImplementation('io.opentelemetry:opentelemetry-sdk-testing')
+
+    testImplementation platform("io.opentelemetry:opentelemetry-bom") {
```

Comment:
Need testImplementation because compileOnly doesn't include the implementation which is needed for the test. Not sure if there is a configuration which combines the two since it seems like it would be pretty common to need both.

Reply:
See my example above. The problem here (i think) is that the BOM specifies the versions. If we only want to specify the version we test against, we can just specify it at the `testImplementation("io.opentelemetry:opentelemetry-sdk-testing:1.20.1")` dependency, we don't need the whole BOM for testing. This way, we could in theory test the API we built against, say, 1.16.0 with the implementation of 1.20.1.